code/modules/admin/verbs/playsound.dm

```diff
@@ -225,3 +225,34 @@
 		if(C && C.chatOutput && !C.chatOutput.broken && C.chatOutput.loaded)
 			C.chatOutput.stopMusic()
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Stop All Playing Sounds") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+
+GLOBAL_LIST_INIT(ambience_files, list(
+	'sound/music/area/bath.ogg',
+	'sound/music/area/bog.ogg',
+	'sound/music/area/catacombs.ogg',
+	'sound/music/area/caves.ogg',
+	'sound/music/area/church.ogg',
+	'sound/music/area/decap.ogg',
+	'sound/music/area/dungeon.ogg',
+	'sound/music/area/dwarf.ogg',
+	'sound/music/area/field.ogg',
+	'sound/music/area/forest.ogg',
+	'sound/music/area/magiciantower.ogg',
+	'sound/music/area/manorgarri.ogg',
+	'sound/music/area/sargoth.ogg',
+	'sound/music/area/septimus.ogg',
+	'sound/music/area/sewers.ogg',
+	'sound/music/area/shop.ogg',
+	'sound/music/area/spidercave.ogg',
+	'sound/music/area/towngen.ogg',
+	'sound/music/area/townstreets.ogg',
+	'sound/music/jukeboxes/tav3.ogg'
+	))
```

Comment:
are these actually used? azure peek has a different map with potentially different areas.

Reply:
Fixed it. Now only the sounds that are used.